src/amitaRJun20/SumOfConsecutiveIntegersInString.java

```diff
@@ -0,0 +1,41 @@
+/*Assignment 26 : 9th July 2020
+WAP to sum all the numbers in given string.
+
+Input: P71Rr29T2E
+Ouput: Sum of all numbers in String is: 102
+
+Input: P6Rr29T2E3
+Ouput: Sum of all numbers in String is: 40*/
+package amitaRJun20;
+
+public class SumOfConsecutiveIntegersInString {
+	
+	//method to find sum of all integers in a String
+	int findSumOfIntegers(String input) {
+		System.out.println("\nThe given String is " + input);
+		int sum = 0;
+		String temp = "";
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				temp += String.valueOf(input.charAt(index));
```

Comment:
String.Valueof method is not required here. we can add String + char which will return String.

Reply:
You are right Mahesh...thanks for the feedback..updated the code...